src/witan/workspace/util.clj

```diff
@@ -32,3 +32,22 @@
        (drop 1)
        (butlast)
        (apply str)))
+
+(defn replacer
+  "Calls  replacement function on different types"
+  [rfn x]
+  (condp = (type x)
+    clojure.lang.Keyword (-> x name rfn keyword)
+    clojure.lang.MapEntry (update x 0 (partial replacer rfn))
```

Comment:
It's so you can call `(map hyphen->underscore m)` on a hashmap. The motivation is to not have to interchange between `:kebab-case` and `:snake_case` as we go in and out of Cassandra.